esti/export_hooks_files/delta/azure/_lakefs_actions/delta_export.yaml

```diff
@@ -0,0 +1,30 @@
+name: Delta Exporter
+on:
+  post-commit:
+    branches: ["{{ .Branch }}*"]
+hooks:
+  - id: delta_exporter
+    type: lua
+    properties:
+      script: |
+        local azure = require("azure")
+        local formats = require("formats")
+        local delta_exporter = require("lakefs/catalogexport/delta_exporter")
+        
+        local table_descriptors_path = "_lakefs_tables"
+        local sc = azure.client(args.azure.storage_account, args.azure.access_key)
+        local delta_client = formats.delta_client(args.lakefs.access_key_id, args.lakefs.secret_access_key)
+        local delta_table_locations = delta_exporter.export_delta_log(action, args.table_names, sc.put_object, delta_client, table_descriptors_path)
+
+        for t, loc in pairs(delta_table_locations) do
+          print("Delta Lake exported table \"" .. t .. "\"'s location: " .. loc .. "\n")
+        end
+      args:
+        azure:
+          storage_account: "{{ .AzureStorageAccount }}"
```

Comment:
Is this a secret? Consider just putting this as plaintext here if it isn't

Reply:
It makes it easier to change the storage account name (in a single place) in the code and not in the yaml file